src/components/ExportCombinedPopUp.vue

```diff
@@ -80,130 +90,152 @@ export default {
   },
   methods: {
     async exportMap () {
-      this.exporting = true
-      const eeLayer = this.eeLayer
-      const body = {
-        liwo_ids: eeLayer.metadata.liwo_ids,
-        band: eeLayer.metadata.band,
-        scale: parseFloat(this.exportScale),
-        export: true
-      }
-      const requestOptions = {
-        method: 'POST',
-        mode: 'cors',
-        headers: { 'Content-Type': 'application/json' },
-        body: JSON.stringify(body)
+      if (!this.exportName && !this.formErrors.includes("Schaal is verplicht")) {
+        this.formErrors.push("Schaal is verplicht")
+      } else if (this.exportName) {
+        this.formErrors = []
       }
```

Comment:
Since there is no `exportName` v-model this looks like it should always throw the "Schaal is verplicht"-error, which it indeed does for me. https://deploy-preview-587--liwo-static.netlify.app//#/combined/7/19435,19431/waterdepth I think you meant to use `exportScale`

Reply:
Testing on two typed strings feels a bit brittle, and I would expect the error to be reflected on the input itself as well. If possible I would suggest adding a conditional item near the input field, and giving said field an error state instead.